language-extensions/dotnet-core-CSharp/build/windows/restore-packages.cmd

```diff
@@ -0,0 +1,24 @@
+@ECHO off
```

Comment:
I don't think we should check-in this file. Please see the main readme of our internal repo. Last line under Getting Started.